### DISCUSSION TITLE
[Resolved] Error about letter spacing in latest release

### DISCUSSION THREAD

**Discussion by vkoopmans-resoluut** [ingrammicro/puzzle-tokens#48] [Q&A] [answered]
![Screenshot 2021-10-21 at 11 16 07](https://user-images.githubusercontent.com/40267611/138248669-524157c4-7dee-4a7d-aa92-cb4a9ab28cce.png)

@MaxBazarov With the latest version I get this error about letterspacing. Didnt have that with version 8.17.0
You know what changed, or what I can do to prevent them :)

**Comment by MaxBazarov** [comment]
Ok. Let me check it

**Comment by vkoopmans-resoluut** [comment, marked as answer]
@MaxBazarov Found it. My bad.... missed a closing ';'

**Reply by vkoopmans-resoluut** [reply]
THe older version of the plugin ignored that probably?

**Reply by MaxBazarov** [reply]
Changed handling of token trailing comments.
Could you show me the source line?

**Comment by vkoopmans-resoluut** [comment]
.01_Headings .H3{
	.Center, .Left, .Right, .Center-white{
		font-family: @font-faces[H3];
		font-size: @font-size[xxxl];
		vertical-align: middle;
		line-height: @line-height[xl];
		text-transform: @font-transform[H3];
		letter-spacing: @font-spacing[H3];
	}

**Comment by MaxBazarov** [comment]
So, does it work with fixed closing ';'?

чт, 21 окт. 2021 г. в 12:45, Vincent Koopmans ***@***.***>:

**Reply by vkoopmans-resoluut** [reply]
Yes it's fixed now 👍

**Reply by MaxBazarov** [reply]
Good!
It's a magic how it works before.
